src/compressed_tensors/compressors/sparse_compressors/base.py

```diff
@@ -59,18 +59,27 @@ class BaseSparseCompressor(BaseCompressor):
     :param config: config specifying compression parameters
     """
 
-    def compress(self, model_state: Dict[str, Tensor]) -> Dict[str, Tensor]:
+    def compress(
+        self,
+        model_state: Dict[str, Tensor],
+        compression_targets: Optional[Set[str]] = None,
+    ) -> Dict[str, Tensor]:
         """
         Compresses a dense state dict using bitmask compression
 
         :param model_state: state dict of uncompressed model
+        :param compression_targets: optional set of layer prefixes to compress, if None
+            compress all layers (for backwards compatibility)
```

Comment:
What are we holding backwards compatibility with? Ideally this should default to only compressing models that we detect the 50% sparsity threshold for

Reply:
For older configs, we will not have targets, this handles those cases;

For newer flow, compression_targets will only contain modules that are over 50% sparse once this lands: https://github.com/vllm-project/llm-compressor/pull/822